src/stylesheets/blocks/_header.scss

```diff
@@ -1 +1,143 @@
+.header {
+  background-color: color(mint-green);
+  width: 100%;
+}
+
+.header-wrapper {
+  display: block;
+  margin: 0 auto;
+  max-width: rem(680);
+  position: relative;
+}
+
+.header__background-img {
+  background-repeat: no-repeat; 
+  background-size: cover;
+  display: inline-block;
+  position: absolute;
+}
+
+.img-1 {
+  background-image: url('../../images/Pill_Orange_Title-01.svg');
+  height: rem(75);
+  top: rem(-21);
+  width: rem(75);
+
+  @media (screen(lg)) {
+    height: rem(125);
+    top: rem(-38);
+    width: rem(125);
+  }
+}
+
+.img-2 {
+  background-image: url('../../images/3Pills_Title-01.svg');
+  height: rem(120);
+  right: rem(-70);
+  top: rem(-10);
+  width: rem(165);
+
+  @media (screen(lg)) {
+    height: rem(180);
+    right: rem(-135);
+    top: rem(-10);
+    width: rem(250);
+  }
+}
+
+.img-3 {
+  background-image: url('../../images/Pills_Purple_Title-01.svg');
+  bottom: rem(-8);
+  height: rem(67);
+  right: rem(-40);
+  width: rem(72);
+
+  @media (screen(lg)) {
+    bottom: rem(15);
+    height: rem(126);
+    right: rem(-150);
+    width: rem(136);
+  }
+}
+
+.img-4 {
+  @media (screen(lg)) {
+    background-image: url('../../images/2Pills_Mint&Beige_Title-01.svg');
+    bottom: rem(180);
+    height: rem(120);
+    left: rem(-160);
+    width: rem(125);
+  }
+}
+
+.header__title {
+  color: color(emerald-green);
+  font-family: $base-med;
+  font-size: rem(11);
+  font-weight: lighter;
+  letter-spacing: rem(2.5);
+  margin: 0;
+  padding: rem(47) 0 rem(10) 0;
+  text-align: center;
+  text-transform: uppercase;
+
+  @media (screen(lg)) {
+    font-size: rem(14);
+    padding: rem(95) 0 rem(10) 0;
+  }
+}
+
+.header__img {
+  box-sizing: border-box;
+  display: block;
```

Comment:
these are a few more styles that I typically apply at the reset level; border-box to all elements, block to images. also common to give `width: 100%; max-width: 100%;` to images as well